frontend/src/components/shared/Banner/Banner.js

```diff
@@ -1,44 +1,43 @@
-import React, { useEffect, useState } from "react";
-import Icon from "../../Icon/Icon";
-import SearchBanner from "../SearchBanner/SearchBanner";
-import MovieDetailsBanner from "../MovieDetailsBanner/MovieDetailsBanner";
-import {
-  BannerContainerSearch,
-  BannerContainerMovieDetails,
-} from "./BannerStyles";
-import movies from "../../../Data";
-
-export default function Banner({
-  movieModal,
-  addMovieModal = { addMovieModal },
-  closeModal,
-  switchBanner,
-  setSwitchBanner,
-  movieDetailsId,
-  modalTitle,
-  setMovieModal,
-}) {
-  if (switchBanner) {
-    return (
-      <BannerContainerMovieDetails>
-        <Icon setSwitchBanner={setSwitchBanner} />
-        <MovieDetailsBanner
-          setSwitchBanner={setSwitchBanner}
-          movieDetailsId={movieDetailsId}
-        />
-      </BannerContainerMovieDetails>
-    );
-  }
-  return (
-    <BannerContainerSearch>
-      <Icon setSwitchBanner={setSwitchBanner} />
-      <SearchBanner
-        setMovieModal={setMovieModal}
-        modalTitle={modalTitle}
-        movieModal={movieModal}
-        addMovieModal={addMovieModal}
-        closeModal={closeModal}
-      />
-    </BannerContainerSearch>
-  );
-}
+import React from "react";
+import Icon from "../Icon/Icon";
+import SearchBanner from "../../Banners/SearchBanner/SearchBanner";
```

Comment:
As you can see here, you have to repeat component name when importing it anywhere. For your convinience, please, in every component folder create file index.js with lines like that
```
import ComponentName from './ComponentName';
export default ComponentName;
```

so you'd be able to import all components like that
```suggestion
import SearchBanner from "../../Banners/SearchBanner";
```

Reply:
I think I disagree with this, I dont think some files should be index.js and others named the component name. I think they should all be named what the component actually is, if I have multiple index.js files open in vs code this will be confusing